### PR TITLE
[dev][cleanup]Code clean up within project views

### PR DESCRIPTION
**Pull request by HarryRybacki**
## Purpose
I noted inconsistencies in the naming conventions for views. As per @sloria's suggestion -- I've started cleaning them up.

## Changes
More consistent return variable names

## Side Effects
Easier to read code?

### CODE REVIEW

File: website/project/views/wiki.py [hunk 1]
```diff
@@ -0,0 +1 @@
+x`
```
Comment (sloria) [inline website/project/views/wiki.py]: This is causing the build to fail. Can you remove this?

Comment (HarryRybacki) [inline website/project/views/wiki.py]: Resubmitted. I ran tests after the changes locally -- weird. Sorry about
that.

On Sat, Feb 14, 2015 at 3:14 PM, Steven Loria <notifications@github.com>
wrote:

> In website/project/views/wiki.py
> <https://github.com/CenterForOpenScience/osf.io/pull/1801#discussion_r24716989>
> :
>
> > @@ -0,0 +1 @@
> > +x`
>
> This is causing the build to fail. Can you remove this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/1801/files#r24716989>
> .
>



-- 
*Harry Rybacki* - Developer, Center for Open Science
Research: http://scinet.osf.io
Blog: http://rybacki.io
[image: http://github.com/harryrybacki]
<http://github.com/HarryRybacki> [image:
https://twitter.com/HarryRybacki] <https://twitter.com/HarryRybacki> [image:
http://www.linkedin.com/in/HarryRybacki/]
<http://www.linkedin.com/in/HarryRybacki/> [image:
https://plus.google.com/+HarryRybacki]
<https://plus.google.com/+HarryRybacki>

Comment (sloria) [inline website/project/views/wiki.py]: There's still a stray backtick here.

Comment (HarryRybacki) [inline website/project/views/wiki.py]: wtf is going on with my local branch when I push? Headed to the office in a
minute -- will re-look upon arrival


On Sun, Feb 15, 2015 at 1:16 PM, Steven Loria <notifications@github.com>
wrote:

> In website/project/views/wiki.py
> <https://github.com/CenterForOpenScience/osf.io/pull/1801#discussion_r24724538>
> :
>
> > @@ -0,0 +1 @@
> > +x`
>
> There's still a stray backtick here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/1801/files#r24724538>
> .
>



-- 
*Harry Rybacki* - Developer, Center for Open Science
Research: http://scinet.osf.io
Blog: http://rybacki.io
[image: http://github.com/harryrybacki]
<http://github.com/HarryRybacki> [image:
https://twitter.com/HarryRybacki] <https://twitter.com/HarryRybacki> [image:
http://www.linkedin.com/in/HarryRybacki/]
<http://www.linkedin.com/in/HarryRybacki/> [image:
https://plus.google.com/+HarryRybacki]
<https://plus.google.com/+HarryRybacki>

**Pull request closed without merging**